gemfiles/4.2.gemfile

```diff
@@ -2,9 +2,9 @@
 
 source "https://rubygems.org"
 
+gem "activerecord", "~> 4.2.5.1"
```

Comment:
The files in `gemfiles/` are generated by Appraisal, so we shouldn't edit them. Instead, let's tell RuboCop to exclude this directory.